providers/inline-evaluating-provider/src/main/java/dev/openfeature/contrib/providers/inlineeval/InlineEvaluatingProvider.java

```diff
@@ -0,0 +1,75 @@
+package dev.openfeature.contrib.providers.inlineeval;
+
+import dev.openfeature.sdk.*;
+import io.github.jamsesso.jsonlogic.JsonLogic;
+import io.github.jamsesso.jsonlogic.JsonLogicException;
+
+import java.util.function.Function;
+
+public class InlineEvaluatingProvider implements FeatureProvider {
+    private final JsonLogic logic;
+    private final RuleFetcher fetcher;
+
+    void initialize(EvaluationContext initialContext) {
```

Comment:
Why do we need to this non public method here? can we delegate the responsibility to an application author when they create an instance of RuleFetcher?

Reply:
Should be public. This is in an ofep currently, but this provider will need to do some bootstrapping as part of it's lifecycle.